toolkits/springer/packages/springer-listing/scss/10-settings/_listing.scss

```diff
@@ -1,17 +1,8 @@
-$listing--font-family: $context--font-family-sans;
-$listing--title-font-family: $context--font-family-serif;
-$listing--title-font-size: 1.125rem;
-$listing--title-font-weight: normal;
-$listing--title-margin-bottom: 4px;
-$listing--title-line-height: 1.3;
-$listing--text-column-flex: 1 0 70%;
-$listing--text-font-size: 1rem;
```

Comment:
Out of interest what was the thought behind removing font size from the variables and 50-component CSS? On our pages such as https://www.springer.com/journal/4/updates where we use this component we will need to set font-size: 1rem in our application css. Which is totally fine! I just wanted to ask and get onboard with your thinking. I guess setting font-size in this component prevents re-use on other Springer products where the base font-size is different? Makes sense I guess

Reply:
@benjclark Good question. I removed it because I don't see these kinds of basic styles set as variables in any other components. `1rem` should be the default, right? It seemed to me, brand context styles were needlessly creeping into components. But, sorry, I may have misunderstood that. The cards do appear to render in the demos correctly.

Reply:
![A scene from Batman Returns where the Bat Signal lights a study in the new Wayne Manor and Bruce Wayne rises from their chair and looks towards it](https://media.giphy.com/media/oOK9AZGnf9b0c/giphy.gif)


The recent changes should mean that:

`html` is set at 100%
`body` is set at the `rem` equivalent of what was there before
the ruleset that had a `font-size` of `13px` on the body, before being set again is removed.

So this is all good tp me -- and with use `rem` it means that with a body `font-size` of `1.125rem` any child element that is set as `1rem` would be `16px` rather than `18px` which would happen if we used the `em` unit.

Reply:
FWIW we might be absolutely fine with two springer.com domain websites being divergent in the font-size they use for this component.

And looking now at the springer.com website I help look after we appear to have two different listing components (fail) that use two different font-sizes (utter fail, ffs). 

So forget about this for now. I don't want to slow this PR down loads by blocking it with this consistency discussion. 

If you could send me a link to a page that has implemented `c-listings` at some point in the future that would be great. Then I could start a discussion with FJ and James Alegria etc and show them all the different listing components and their font-sizes across our Springer domain websites and they can make a call on if they should all have the same font-size or not for a consistent UX and also if they should all be the same component.